include/sddf/i2c/devices/ds3231/ds3231.h

```diff
@@ -64,14 +64,14 @@ enum ds3231_registers {                         // RANGE:
 #define DEFAULT_READ_RESPONSE_RETRIES (256)
 #define DEFAULT_READ_ACK_FRAME_RETRIES (20)
 
-bool ds3231_write(uint8_t *buffer, uint8_t buffer_len, size_t retries);
-bool ds3231_read(uint8_t *buffer, uint8_t buffer_len, size_t retries);
+int ds3231_write(uint8_t *buffer, uint16_t buffer_len, size_t retries);
+int ds3231_read(uint8_t *buffer, i2c_addr_t reg_addr, uint16_t buffer_len, size_t retries);
 
 // User Provided Functions
-bool ds3231_get_time(uint8_t *seconds, uint8_t *minutes, uint8_t *hours, uint8_t *day_of_week, uint8_t *day,
-                     uint8_t *month, uint8_t *year);
-bool ds3231_set_time(uint8_t seconds, uint8_t minutes, uint8_t hours, uint8_t day_of_week, uint8_t day, uint8_t month,
-                     uint8_t year);
+int ds3231_get_time(uint8_t *seconds, uint8_t *minutes, uint8_t *hours, uint8_t *day_of_week, uint8_t *day,
```

Comment:
Why change the return type?

Reply:
To make return typing consistent with the other ds3231 functions